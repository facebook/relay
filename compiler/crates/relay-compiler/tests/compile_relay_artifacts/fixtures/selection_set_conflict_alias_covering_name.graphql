# expected-to-throw
query selectionSetConflictAliasCoveringNameQuery {
  me {
    name
    name: canViewerComment
  }
}
