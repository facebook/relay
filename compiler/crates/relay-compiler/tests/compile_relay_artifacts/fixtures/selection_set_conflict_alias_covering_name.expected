==================================== INPUT ====================================
# expected-to-throw
query selectionSetConflictAliasCoveringNameQuery {
  me {
    name
    name: canViewerComment
  }
}
==================================== ERROR ====================================
✖︎ Field 'name' is ambiguous because it references two different fields: 'name' and 'canViewerComment'

  selection_set_conflict_alias_covering_name.graphql:4:5
    3 │   me {
    4 │     name
      │     ^^^^
    5 │     name: canViewerComment

  ℹ︎ the other field

  selection_set_conflict_alias_covering_name.graphql:5:11
    4 │     name
    5 │     name: canViewerComment
      │           ^^^^^^^^^^^^^^^^
    6 │   }
